mailbox/store/src/main/java/org/apache/james/mailbox/store/StoreMessageManager.java

```diff
@@ -409,6 +423,15 @@ public Long execute() throws MailboxException {
 
     }
 
+    private List<Attachment> parseAttachments(SharedFileInputStream contentIn) {
```

Comment:
I don't think it parses attachment, it parses an email and extract some Attachment